chandra.time/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set data = load_setup_py_data() %}
```

Comment:
It looks like this doesn't work on Py3, I think because load_setup_py_data is trying to JSON serialize the cythonize portion of the setup
```
      ext_modules=cythonize(extensions),
```
Not sure why this seems to work on Py2.  A work-around for this is just to specify the version until we patch load_setup_py_data().